helpers/database.py

```diff
@@ -20,13 +20,19 @@ def setup_db(conn):
         c.execute(
             """CREATE TABLE IF NOT EXISTS votes_history (id INTEGER AUTO_INCREMENT PRIMARY KEY, user_id INTEGER, message_id INTEGER, backer INTEGER, vote_time DATETIME DEFAULT CURRENT_TIMESTAMP)"""
         )
+        c.execute(
+            """CREATE UNIQUE INDEX IF NOT EXISTS one_ring_rules_them_all ON votes_history(message_id, backer)"""
+        )
```

Comment:
Instead of Duplicating the same code isn't it better to write it after the else?

Reply:
I'd leave it this way. Makes it clear that they are different sql languages. Will also be easier if something needs to change in the future that has different Syntax